inspirehep/dojson/institutions/fields/bd1xx.py

```diff
@@ -32,20 +36,15 @@
 @utils.filter_values
 def location(self, key, value):
     """GPS location info."""
-    longitude, latitude = ('', '')
-    if value.get('d'):
+    def _get_float(value, c):
         try:
-            longitude = float(value.get('d'))
-        except:
-            pass
-    if value.get('f'):
-        try:
-            latitude = float(value.get('f'))
-        except:
-            pass
+            return float(value[c])
+        except (KeyError, ValueError):
+            return ''
+
     return {
-        'longitude': longitude,
-        'latitude': latitude
+        'longitude': _get_float(value, 'd'),
```

Comment:
Then they will be skipped, as documented by the tests: https://github.com/jacquerie/inspire-next/blob/ff5b8b2be01c6386973f17ae2c587a9664ccac54/tests/unit/dojson/test_dojson_institutions.py#L80-L90